libsemanage/tests/test_iface.c

```diff
@@ -546,6 +567,8 @@ void test_iface_modify_del_query_local(void)
 	CU_ASSERT(semanage_iface_query_local(sh, key, &iface_local) < 0);
 
 	/* cleanup */
+	semanage_iface_key_free(key);
+	semanage_iface_free(iface_local);
```

Comment:
Same comment as https://github.com/SELinuxProject/selinux/pull/321/commits/322243ee7d34f1b3e23c04c75a6b5c4f597092f7#r730306247 here: `semanage_iface_free(iface_local);` should be moved up a few lines, before `CU_ASSERT(semanage_iface_query_local(sh, key, &iface_local) < 0);`